File: JavaBasic/116_product_digits.java
// 116	116_product_digits.java	Find the product of all digits of a given number.
public class _116_product_digits {
    public static void main(String[] args) {
        int num = 234, product = 1, temp = num;
        while (temp > 0) { product *= temp%10; temp /= 10; }
        System.out.println("Number: " + num + " | Product of digits: " + product);
    }
}

File: JSBASIC/116_product_digits.js
// 116	116_product_digits.js	Find the product of all digits of a given number.

let num = 234;
let product = 1;
let temp = num;
while (temp > 0) {
    product *= temp % 10;
    temp = Math.floor(temp / 10);
}
console.log("Number:", num);
console.log("Product of digits:", product);
